app/controllers/saml_idp_controller.rb

```diff
@@ -58,7 +58,9 @@ def render_template_for(message, action_url, type)
   end
 
   def apply_secure_headers_override
-    use_secure_headers_override(:saml)
+    override_content_security_policy_directives(
+      form_action: ["'self'", *SecureHeadersWhitelister.whitelisted_domains]
```

Comment:
Sure, I can do that, but the test for the update for example made sure the list of domains was changed. I'll push a commit to show what that looks like